### PR TITLE
feature: add close tabs to the right, #822

### PR DESCRIPTION
**Pull request by Alvoda**
Please let me know if this is ok. If something is missing just tell me.
As far as i can see this works just fine. 
I have no automated test for this. Is this required?

### CODE REVIEW

**Comment by mlangkabel** [comment]
Thank you for handing in this pull request! The failed CI build is probably not your fault. I'm looking into that right now.

**Comment by Alvoda** [comment]
Did you get a chance to look into it?
Is there anything i could do to help?

**Comment by egraether** [comment]
I'm looking into it now.

File: src/lib_gui/qt/element/QtTabBar.cpp [hunk 1]
```diff
@@ -1,4 +1,8 @@
 #include "QtTabBar.h"
+#include <QtContextMenu.h>
+#include <QtGraphicsView.h>
+#include <logging.h>
+#include <QContextMenuEvent>
```
Comment (egraether) [inline src/lib_gui/qt/element/QtTabBar.cpp]: Please follow convention:
For Qt headers we don't add '.h' e.g. `#include <QContextMenuEvent>`
For own headers we use double quotations e.g. `#include "QtContextMenu.h"`
Qt headers are placed above own headers (see other files)

File: src/lib_gui/qt/graphics/base/QtGraphicsView.h [hunk 2]
```diff
@@ -76,6 +76,7 @@ private slots:
 
 	void legendClicked();
 
+
```
Comment (egraether) [inline src/lib_gui/qt/graphics/base/QtGraphicsView.h]: Please remove this change.

File: src/lib_gui/qt/element/QtTabBar.cpp [hunk 3]
```diff
@@ -19,3 +23,22 @@ QSize QtTabBar::minimumTabSizeHint(int index) const
 {
 	return QSize(45, QTabBar::minimumTabSizeHint(index).height());
 }
+
+void QtTabBar::contextMenuEvent(QContextMenuEvent* event)
+{
+    QtContextMenu menu(event, this);
```
Comment (egraether) [inline src/lib_gui/qt/element/QtTabBar.cpp]: Your whole implementation uses the wrong indentation. Please change everything to `tabs` instead of `spaces`.

**Comment by Alvoda** [comment]
thanks for the feedback. will make the changes and then turn on to the next issue...

**Comment by mlangkabel** [comment]
Yes, I managed to fix the CI. So once you made the changes @egraether requested, you can make a new commit and the CI will automatically re-run the checks on your pull request.

**Comment by Alvoda** [comment]
So, i hope this is it :)
Have been struggling with KDevelop. I guess i'll switch back to vim. Much less trouble...

**Comment by egraether** [comment]
Thanks!

**Pull request merged**